ui/lib/core/addon/utils/capabilities.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const PATH_MAP = {
+  customMessages: 'sys/config/ui/custom-messages',
+};
```

Comment:
I added this while migrating custom messages since capabilities are fetched in multiple routes and I thought this would help to avoid typos or other issues by typing out the same path in multiple places.